Dockerfile

```diff
@@ -25,4 +25,7 @@ WORKDIR $NEW_DIR
 SHELL ["/bin/bash", "-o", "pipefail", "-lc"]
 RUN curl -sSL "$NEWINSTALL_URL" | bash -s -- -cbtS
 
+RUN source ./loadLSST.bash
```

Comment:
This shouldn't work, as the two RUN commands invoke separate shells, so the results of the `source` are unavailable to the subsequent `conda install`.

I was actually thinking of perhaps having a separate Dockerfile altogether that would build on the standard Science Pipelines container allowing any needed injections for PanDA purposes.

Reply:
Hi @ktlim  Please help point to the github repo for the standard Science Pipeline container. Thanks.